controllers/apply/lib/field-types/checkbox.js

```diff
@@ -29,7 +29,7 @@ class CheckboxField extends Field {
     defaultSchema() {
         const options = this.options || [];
         const baseSchema = Joi.array()
-            .items(Joi.string().valid(options.map((option) => option.value)))
+            .items(Joi.string().valid(...options.map((option) => option.value)))
```

Comment:
This is the most common change we need to make. `.valid` now accepts a list of values, rather than an array.